extension/tamper.css

```diff
@@ -31,6 +31,7 @@ h1 {
 label {
   display: block;
   padding: 5px 0;
+  color: blue;
```

Comment:
note that it seems this will be configurable in a future version of chrome

https://bugs.chromium.org/p/chromium/issues/detail?id=608869